### ISSUE TITLE
Only 'SDK Info' Command Available

### ISSUE THREAD

**Issue by Aza1** [radicaled/dart-tools#87]
I'm using Atom 0.204.0 on Windows 7, and the only visible command on Packages>Dart Tools is SDK Info.

Also saving my pub.yaml file does not trigger pub, even though the 'Pub Get On Save' option is set.

**Comment by radicaled** [comment]
Can you (CTRL/CMD)+SHIFT+P, open "toggle dev tools", and copy and paste any stack trace in the console?

**Comment by hectorfhurtado** [comment]
HI @Aza1, I had the same problem. It was because I didn't write where my dart-sdk was on Settings. After that, commands appeared.

Hope that helps

**Comment by radicaled** [comment]
@Aza1 

Are you still having problems? Anything else I can do for you?

**Comment by Aza1** [comment]
@radicaled 
pub get now runs automatically when I change pubspec.yaml, which is great. 
But the only command still available for Dart Tools is SDK Info, none of the others, it shows the correct sdk info by the way.

**Comment by radicaled** [comment]
@Aza1 

OK. If you hit CTRL+SHIFT+P and type "Dart" do you see a list of commands? Does autocomplete work if you start typing in a Dart file? Do you see "analysis server started" when you open a Dart project?

The file menu for "Dart Tools" only contains one entry right now.

**Comment by Aza1** [comment]
@radicaled 
Yes I do see the list of commands when I press CRTL+SHIFT+P and type 'Dart'.
Autocomplete works.
And I do see the 'analysis server started' notification.

Any specific reason why the 'Dart Tools' menu only contains just the one entry?

**Comment by radicaled** [comment]
I don't know anyone who uses the "Packages" menu, so it's more or less neglected. I accept packages that populate it, though!

**Comment by radicaled** [comment]
@Aza1 

Any items you'd like to see in the menu?

**Comment by Aza1** [comment]
pub serve
pub build 
pub get
pub upgrade 
pub deps
pub cache
